style.css

```diff
@@ -0,0 +1,162 @@
+body {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  color: #344563;
+  margin: 0;
+  padding: 0;
+}
+
+.name {
+  font-weight: 700;
+  font-size: 48px;
+  line-height: 60px;
+  color: #091e42;
+}
+
+.mobmenu {
+  display: none;
+}
+
+.logo {
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+}
+
+.blue {
+  color: #6070ff;
+}
+
+.space {
+  margin: 0 5%;
+}
+
+.menu {
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+}
+
+.linkstyle a {
+  text-decoration: none;
+}
+
+.gray {
+  color: gray;
+}
+
+.onhover:hover {
+  text-shadow: 0 0 0.15em #6070ff;
+  opacity: 0.5;
+}
+
+.pressed:active {
+  color: #6070ff;
+}
+
+.logo a:hover {
+  opacity: 0.5;
+}
+
+.round {
+  border-bottom-left-radius: 80px;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+  background-color: #fff;
+}
+
+/* flex container */
+.toolbar {
+  display: flex;
+  align-items: center;
+  padding-top: 5%;
+}
+
+.toolbar ul {
+  margin-right: 5%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.liststyle {
+  list-style: none;
+}
+
+/* flex item */
+.toolbar li {
+  flex: 1;
+  margin: 0 10px;
+}
+
+.toolbar div {
+  flex: 2;
+}
+
+.wide {
+  max-width: 920px;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  flex: 1;
+  justify-content: center;
+  align-items: center;
+  background-color: #fff;
+  background-image: url('Img/Headerbg@2x.png');
```

Comment:
- [ ] As requested by the previous reviewer please fix the background image and make sure it matches with the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1 )template. You are using the background image that should be used for only desktop screens. Please download the background image for mobile screens and use that for this project. (You can download images from Figma select the element from the left sidebar and export it from right properties options.)

### Your design

![image](https://user-images.githubusercontent.com/83803414/164616921-048b3981-8159-45f7-b2da-de08d6516fad.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/164617120-b01c9ce4-751f-4beb-a0c2-f8e577b281b5.png)